data/data-pipeline/data_pipeline/tests/sources/example/test_etl.py

```diff
@@ -30,6 +29,14 @@ class TestETL:
     _TRANSFORM_CSV_FILE_NAME = "transform.csv"
     _OUTPUT_CSV_FILE_NAME = "output.csv"
 
+    # This *does* need to be updated in the child class. It specifies where the "sample data" is
```

Comment:
This is really inelegant. One annoying thing about these tests is you have to "set them up" with sample data. I thought it would be nice if we could only put the sample data in one place -- the data folder for the test -- and then update the "sample data" for the transform step using the same `--snapshot-update` flag. A key issue here, though, is that to do this, we have to read the CSV in and pandas will change some types (not values). Do we think this is a problem / do you have any thoughts @widal001?

Reply:
One other note -- this is because the snapshot-update requires string/bytes as a comparison.

Reply:
@widal001 this was an intentional design choice, but a very silly one! In our existing workflow, the sample data "lives" somewhere else. I also think -- but might be misremembering -- that this is necessary for when we load / transform the "actual" data. I like your idea a lot, though, and think we should implement it!